packages/dart/lib/src/network/parse_websocket_io.dart

```diff
@@ -8,10 +8,10 @@ import 'package:web_socket_channel/web_socket_channel.dart';
 class WebSocket {
   WebSocket._(this._webSocket);
 
-  static const int CONNECTING = 0;
-  static const int OPEN = 1;
-  static const int CLOSING = 2;
-  static const int CLOSED = 3;
+  static const int connecting = 0;
```

Comment:
Is this only used internally? If not, could this be a breaking change?

Reply:
It is internal only and not a breaking change